lib/utilities/lint-fix.js

```diff
@@ -1,26 +1,27 @@
 'use strict';
 
-const fs = require('fs-extra');
 const execa = require('../utilities/execa');
+const isYarnProject = require('../utilities/is-yarn-project');
+const prependEmoji = require('../utilities/prepend-emoji');
 
-async function run(ui) {
+async function run(project) {
```

Comment:
Passing in the project allows us to check the presence of the fix script via `project.pkg.scripts` instead of having to read the package file again. It also allows us to use `isYarnProject` instead of check the current directory for a `yarn.lock` file, which I assume didn't work properly when used in a Yarn workspace?